src/aics-image-viewer/components/App/index.tsx

```diff
@@ -339,8 +331,9 @@ const App: React.FC<AppProps> = (props) => {
       changeChannelSetting(channelIndex, "controlPoints", newControlPoints);
     }
     view3d.updateLuts(aimg);
+    view3d.onVolumeData(aimg, [channelIndex]);
```

Comment:
The solution to all the problems I described earlier was to add this line. The test app in volume-viewer makes a similar call, and it seemed to be the only thing missing that could account for that app not having problems where this one did. I'm thrilled to have this fixed, but this seems like a slightly bizarre API choice? Feels like we're doing `view3d`'s work for it...

Reply:
The example viewer in volume-viewer seems to call this in its own onChannelDataArrived, so it's consistent with that, at least.

Reply:
It's a call to update the VolumeDrawable, and it does seem like we could connect that callback internally inside volume-viewer if we wanted to, and if the order of operations was correct.